src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -12,19 +1,60 @@
+type Props = {
+  setStatus: (status: () => string) => void;
+  setQuery: (status: () => string) => void;
+  query: string;
+};
 
-    <p className="control is-expanded has-icons-left has-icons-right">
-      <input
-        data-cy="searchInput"
-        type="text"
-        className="input"
-        placeholder="Search..."
-      />
-      <span className="icon is-left">
-        <i className="fas fa-magnifying-glass" />
-      </span>
+export const TodoFilter: React.FC<Props> = ({ query, setStatus, setQuery }) => {
+  const handlerOnChangeStatus = (
+    event: React.ChangeEvent<HTMLSelectElement>,
+  ): void => {
+    setStatus(() => event.target.value);
```

Comment:
Pass the value directly to `setStatus` instead of using a function. Use `setStatus(event.target.value);`.